dot/rpc/helpers.go

```diff
@@ -44,3 +49,47 @@ func LocalRequestOnly(r *rpc.RequestInfo, i interface{}) error {
 	}
 	return errors.New("external HTTP request refused")
 }
+
+func SnakeCaseFormat(method string) (string, error) {
```

Comment:
This function doesn't need to be public.
`snakeCaseFormat`

Reply:
Done!

Reply:
done